onionconfig/config.py

```diff
@@ -61,6 +66,11 @@ def update(self, module_name):
         if os.path.isfile(dir_):
             dir_ = os.path.dirname(dir_)
         self.directory = dir_
+        if os.path.isfile(os.path.join(self.directory, self.CFG_LMOD_STATS_FILE)):
+            try:
+                self.cfg_lmod = json.load(open(os.path.join(self.directory, self.CFG_LMOD_STATS_FILE), 'r'))
+            except:
+                logger.error('Unable to load cfg lmods for dir %s' % self.directory)
```

Comment:
Replacing the % with a comma (i.e. passing the rest as arguments) lets sentry group messages more effectively